src/lib/PhpCsFixer/Sets/Ibexa50RuleSet.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\CodeStyle\PhpCsFixer\Sets;
+
+final class Ibexa50RuleSet extends AbstractIbexaRuleSet
+{
+    public function getRules(): array
+    {
+        return array_merge(
+            [
+                '@PER-CS2.0' => true,
```

Comment:
disclaimer: what we want or don't want to use from this set can change, I need to see the result of this on our repos.

Reply:
The primary change will affect trailing commas (will enforce trailing comma everywhere where language allows it) and import grouping (`use <function>` and `use <class>` are expected to be in blocks) separated by a single newline, and ordered alphabetically within these groups.

Since we are enforcing a specific PHP-CS-Fixer version, these will in practice be always the same until we change the version of the fixer.

Reply:
> The primary change will affect trailing commas (will enforce trailing comma everywhere where language allows it) and import grouping (`use <function>` and `use <class>` are expected to be in blocks) separated by a single newline, and ordered alphabetically within these groups.
> 
> Since we are enforcing a specific PHP-CS-Fixer version, these will in practice be always the same until we change the version of the fixer.

The other issue I've found was
```
-class InvalidSearchEngine extends InvalidArgumentException
-{
-}
+class InvalidSearchEngine extends InvalidArgumentException {}
```

Asked internally and looks like this one is fine too with PHP Team. So for me this one is also resolved.